File: Demo-LLP-MIP/Demo-LLP-MIP-Python/v1/ipay/argument/refund/RefundQueryParams.py
class RefundQueryParams:
    def __init__(self):
        self.mch_id = None
        self.refund_seqno = None
        self.platform_refundno = None
        self.refund_date = None

    def to_dict(self):
        data = {
            "mch_id": self.mch_id,
            "refund_seqno": self.refund_seqno,
            "platform_refundno": self.platform_refundno,
            "refund_date": self.refund_date
        }
        return {k: v for k, v in data.items() if v}

File: Demo-LLP-MIP/Demo-LLP-MIP-PHP/v1/ipay/argument/refund/RefundQueryParams.php
<?php


class RefundQueryParams
{
    public $mch_id;
    public $refund_seqno;
    public $platform_refundno;
    public $refund_date;

    public function __construct()
    {
        $this->mch_id = null;
        $this->refund_seqno = null;
        $this->platform_refundno = null;
        $this->refund_date = null;
    }

    public function to_dict() {
        $data = [];
        foreach ($this as $key => $value) {
            if ($value !== null) {
                $data[$key] = $value;
            }
        }
        return json_encode($data);
    }
}